fastapi_sqla/sqla.py

```diff
@@ -119,11 +108,12 @@ async def add_session_to_request(request: Request, call_next):
         fastapi_sqla.setup(app)  # includes middleware
 
         @app.get("/users")
-        def get_users(session: fastapi_sqla.Session = Depends()):
+        def get_users(session: fastapi_sqla.Session):
             return session.execute(...) # use your session here
     """
-    async with contextmanager_in_threadpool(open_session()) as session:
-        request.scope[_SESSION_KEY] = session
+    async with contextmanager_in_threadpool(open_session(key)) as session:
+        # TODO: This should use request.state
```

Comment:
Is this for a future PR?

Reply:
Fixed ✅

Reply:
Broken you mean 😅 ?

Reply:
oops, I talked too fast 😂

Reply:
Alright, fixed for real this time 😄 That's what happens when I do thee things at the same time 😅